examples/radio/radio-rating.html

```diff
@@ -0,0 +1,290 @@
+          <div id="ex1">
+            <div class="rating-radio" role="radiogroup" aria-labelledby="id-rating-label" data-rating-value="">
+              <div id="id-rating-label" class="label">Rating</div>
+              <svg role="none" xmlns="http://www.w3.org/2000/svg" width="264" height="48">
+                <defs>
+                  <g id="star">
+                    <polygon points="2.0,13.4 11.7,20.5 8.0,31.1 17.7,24.8 27.4,31.9 23.7,20.5 33.4,13.4 21.4,13.4 17.7,2.0 14.0,13.4"/>
+                  </g>
+                  <g id="focus-ring">
+                    <rect width="40" height="38" rx="4"/>
+                  </g>
+                </defs>
+
+                <g role="radio" aria-checked="false" tabindex="-1" data-rating="1" aria-label="one star" class="star-1 star-2 star-3 star-4 star-5" >
+                  <use class="star" xlink:href="#star" x="10" y="7"/>
+                  <use class="focus-ring" xlink:href="#focus-ring" x="8" y="5"/>
+                </g>
+
+                <g role="radio" aria-checked="false" tabindex="-1" data-rating="2" aria-label="two stars" class="star-2 star-3 star-4 star-5">
+                  <use class="star" xlink:href="#star" x="50" y="7"/>
+                  <use class="focus-ring" xlink:href="#focus-ring" x="48" y="5"/>
+                </g>
+
+                <g role="radio" aria-checked="false" tabindex="-1" data-rating="3" aria-label="three stars" class="star-3 star-4 star-5">
+                  <use class="star" xlink:href="#star" x="90" y="7"/>
+                  <use class="focus-ring" xlink:href="#focus-ring" x="88" y="5"/>
+                </g>
+
+                <g role="radio" aria-checked="false" tabindex="-1" data-rating="4" aria-label="four stars" class="star-4 star-5">
+                  <use class="star" xlink:href="#star" x="130" y="7"/>
+                  <use class="focus-ring" xlink:href="#focus-ring" x="128" y="5"/>
+                </g>
+
+                <g role="radio" aria-checked="false" tabindex="-1" data-rating="5" aria-label="five stars" class="star-5">
+                  <use class="star" xlink:href="#star" x="170" y="7"/>
+                  <use class="focus-ring" xlink:href="#focus-ring" x="168" y="5"/>
+                </g>
+              </svg>
+            </div>
+          </div>
+          <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    </section>
+
+    <section>
+      <h2>Accessibility Features</h2>
+      <ul>
+        <li>
+          To highlight the interactive nature of the rating stars a focus ring appears around the star icon that has focus.  When a pointer hovers over a star the cursor changes to a pointer and the number of filled stars is updated to include the hovered star.  When a pointer leaves a star, the filled stars are restored to indicate the currently checked star.
```

Comment:
>To highlight the interactive nature of the rating stars a focus ring appears around the star icon that has focus.

I don't understand the purpose of this sentence. It sounds like a focus ring appears on the item with focus. That goes without saying; it wouldn't be a special feature to provide a focus indicator.


>When a pointer hovers over a star the cursor changes to a pointer and the number of filled stars is updated to include the hovered star.  When a pointer leaves a star, the filled stars are restored to indicate the currently checked star.

This sounds like a potential problem because the user would not be able to distinguish between hover and selection. Someone using a mouse to drive magnification might think five stars are checked just because they look checked when the mouse puts their mag view over the fifth star. Do I understand this correctly?

Reply:
This is the way other rating stars work that I have found, basically hover fills in stars as if they were the current state.  Not sure if there is any other good way to distinguish between focus and hover state that does not get visually confusing and also works in high contrast mode.  This is similar tot he issue Siri brought up related to  hover and focus styling that recently came up for the disclosure menu example.

Reply:
@mcking65 I have updated the description to talk about how changing hover and changing focus update the number of stars that are filled.  So if focus is moving then the hover pointer is essentially being ignored.  If hover is changing the  number of stars filled changes based on the which star is being hovered.  Selection of the radio is done through a click or a tap.   If the hover moves off the stars it reverts to filling the stars based on the radio that is checked.  I think this makes the most sense, but I am open to any other ideas people have.